src/itest/kotlin/nl/lifely/zac/itest/KlantRestServiceTest.kt

```diff
@@ -1,5 +1,5 @@
 /*
- * SPDX-FileCopyrightText: 2023 Lifely
+ * SPDX-FileCopyrightText: 2023-2024 Lifely
```

Comment:
```suggestion
 * SPDX-FileCopyrightText: 2023 Lifely
```

From our CONTRIBUTING.md:

> where we use the convention that the year indicates the initial year when a contribution was made by that contributor.

Otherwise it becomes unmanageable and does not add anything anyway.

Reply:
Makes sense. Changed